tests/e2e/postgresql/scripts/trigger-maintenance.sh

```diff
@@ -0,0 +1 @@
+../../../e2e/common/scripts/trigger-maintenance.sh
```

Comment:
I wouldn't put the script in the common folder. This script is very PostgreSQL specific.